frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -28,135 +28,44 @@ const getAssessments = (assessment: AssessmentTypes) => {
-    grade: Grade.GRADE_4,
-    type: UseCase.BEGINNING,
-    country: "Canada",
-    region: "Quebec",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 7 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_7,
-    type: UseCase.BEGINNING,
-    country: "USA",
-    region: "California",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 5 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_5,
-    type: UseCase.BEGINNING,
-    country: "Canada",
-    region: "Ontario",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 7 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_7,
-    type: UseCase.BEGINNING,
-    country: "USA",
-    region: "California",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 5 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_5,
-    type: UseCase.BEGINNING,
-    country: "Canada",
-    region: "Ontario",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 7 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_7,
-    type: UseCase.END,
-    country: "USA",
-    region: "California",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 5 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_5,
-    type: UseCase.END,
-    country: "Canada",
-    region: "Ontario",
-  },
-  {
-    status: Status.DRAFT,
-    name: "Grade 7 Ontario Pre-Term Assessment 2016",
-    grade: Grade.GRADE_7,
-    type: UseCase.END,
-    country: "USA",
-    region: "California",
-  },
-];
-
 const DisplayAssessmentsPage = (): React.ReactElement => {
   const unselectedColor = "#727278";
   const [search, setSearch] = React.useState("");
   const [sortProperty, setSortProperty] = React.useState("name");
   const [sortOrder, setSortOrder] = React.useState("ascending");
 
-  // const { loading, error, data } = useQuery(GET_USERS_BY_ROLE, {
-  //   fetchPolicy: "cache-and-network",
-  //   variables: { role: "Admin" },
-  // });
+  const { loading, error, data } = useQuery(GET_ALL_TESTS, {
+    fetchPolicy: "cache-and-network",
+    variables: { role: "Admin" },
+  });
 
   const searchedAssessements = React.useMemo(() => {
-    let filteredTests = sampleAssessments;
+    if (!data) return [];
+    let filteredTests = data.tests;
+
     if (search) {
       filteredTests = filteredTests.filter(
         (assessment: AssessmentTypes) =>
           assessment.grade.toLowerCase().includes(search.toLowerCase()) ||
-          assessment.country.toLowerCase().includes(search.toLowerCase()) ||
-          assessment.region.toLowerCase().includes(search.toLowerCase()) ||
-          assessment.type.toLowerCase().includes(search.toLowerCase()),
+          assessment.curriculumCountry
+            .toLowerCase()
+            .includes(search.toLowerCase()) ||
+          assessment.curriculumRegion
+            .toLowerCase()
+            .includes(search.toLowerCase()) ||
+          assessment.assessmentType
+            .toLowerCase()
+            .includes(search.toLowerCase()),
```

Comment:
Yup I already talked to design and it's supposed to include those four specific fields, the same one's used in the Filter Menu

Reply:
okay! but should it include search for `Name` as well? i noticed it's missing!

additionally, do search queries work for the `grades` field? because it is an enum and its value is `GRADE_X`, i think it only matches queries when they follow that format
i.e to find `Grade 6` queries, searching `6` or `Grade 6` doesn't work, only `GRADE_6` or `grade_6`

Reply:
yeah design didn't mention searching by name but I'll add that because that very intuitive

Reply:
Add name searching, and changed grade searching so it searches without underscores as well as only for the number!